commands/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Sample Commands Extension",
+  "description": "Press Ctrl+Shift+Y to send an event (Command+Shift+Y on a Mac).",
+  "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/commands",
+  "manifest_version": 2,
+  "version": "1.0",
+  "background": {
+    "scripts": ["background.js"]
+  },
+  "applications": {
+    "gecko": {
+      "id": "commands@mozilla.org"
```

Comment:
I think we should include strict_min_version in here, especially for APIs that require newer Firefoxes:

"strict_min_version": "48.0a1"

...should work.